input/system/selfhosted/otel_handler.go

```diff
@@ -28,8 +27,6 @@ import (
 
 func logLineFromJsonlog(record *common.KeyValueList, tz *time.Location) (state.LogLine, *state.LogLine) {
 	var logLine state.LogLine
-	logLine.CollectedAt = time.Now()
-	logLine.UUID = uuid.NewV4()
```

Comment:
These lines don't look like having duplicate. Can you tell me where these are generated? (the same for tembo)

Reply:
See commit message for an explanation - we assign it in the log runner.